joomla-projects/gsoc22_accessibility-plugin-script
# Fixed the italic icons and corrected the emoji for underline links

### Testing instructions

Just check that the close and reset icons are not italic anymore and icon for underline links is correct.
